### PR TITLE
[usecase] Add test for initialize

### PR DESCRIPTION
**Pull request by wanghongjuan**
Impacted TCs num(approved): New 1, Update 0, Delete 0
Unit test Platform: Android
Android test result summary: Pass 1, Fail 0, Block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-4794

### CODE REVIEW

File: usecase/usecase-wrt-android-tests/samples/Initialize/res/videos.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿<!DOCTYPE html> 
+<!-- Copyright © 2015 Michele Placentino,Adriano Perla. All Rights Reserved-->
```
Comment (ibelem) [inline usecase/usecase-wrt-android-tests/samples/Initialize/res/videos.html]: +
@zqzhang Can we leverage this kind of sample as test cases directly in Crosswalk Test Suite?

Comment (zqzhang) [inline usecase/usecase-wrt-android-tests/samples/Initialize/res/videos.html]: No, if just this line. Could you please point me the origin of these files?

**Comment by ibelem** [comment]
@zqzhang The original test case comes from http://www.filedropper.com/adriandemocanary004arm in https://crosswalk-project.org/jira/browse/XWALK-4778 . Any suggestions? I think we can cut down to simple test case rather than demo by ourselves.

@wanghongjuan

**Comment by zqzhang** [comment]
> I think we can cut down to simple test case rather than demo by ourselves.

Yes, this is better.

By the way, I cannot access to http://www.filedropper.com/adriandemocanary004arm. It is blocked by Intel network.

**Comment by wanghongjuan** [comment]
Updated

**Comment by ibelem** [comment]
LGTM

**Pull request merged**